pom.xml

```diff
@@ -3,31 +3,53 @@
     xmlns="http://maven.apache.org/POM/4.0.0"
     xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd">
     <modelVersion>4.0.0</modelVersion>
-    <groupId>net.mekomsolutions</groupId>
-    <artifactId>eip-routes-senaite</artifactId>
+    <groupId>com.ozonehis</groupId>
+    <artifactId>eip-openmrs-senaite-routes</artifactId>
     <version>1.3.0-SNAPSHOT</version>
     <packaging>jar</packaging>
-    <name>Routes to integrate OpenMRS with SENAITE</name>
+    <name>EIP OpenMRS SENAITE Routes</name>
+    <description>Routes to integrate OpenMRS with SENAITE</description>
+
     <properties>
         <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
         <maven.compiler.source>1.8</maven.compiler.source>
         <maven.compiler.target>1.8</maven.compiler.target>
-        <eipClientVersion>1.0.0</eipClientVersion>
+        <eipClientVersion>1.2.0-SNAPSHOT</eipClientVersion>
     </properties>
+
+    <repositories>
+        <repository>
+            <id>openmrs-repo</id>
+            <url>https://mavenrepo.openmrs.org/nexus/content/repositories/public/</url>
+        </repository>
+        <repository>
+            <id>mekom-repo</id>
+            <url>https://nexus.mekomsolutions.net/repository/maven-public/</url>
+        </repository>
+    </repositories>
+
     <dependencies>
         <dependency>
-            <groupId>net.mekomsolutions.eip</groupId>
-            <artifactId>openmrs-eip-client-commons</artifactId>
+            <groupId>com.ozonehis.eip</groupId>
+            <artifactId>eip-client-commons</artifactId>
```

Comment:
For group Id, should we go with just `com.ozonehis` or `com.ozonehis.eip`? we need to be consistent

Reply:
I think the group Id should be `com.ozonehis`, then we add a package (eip) `com.ozonehis.eip.*` for all classes